src/common.browser/PCMRecorder.ts

```diff
@@ -107,7 +107,6 @@ export class PcmRecorder implements IRecorder {
             }
             if (this.privMediaResources.source) {
                 this.privMediaResources.source.disconnect();
-                this.privMediaResources.stream.getTracks().forEach((track: any) => track.stop());
```

Comment:
>his.privMediaResources.stream.getTracks().forEach((track: any) => track.stop()); [](http://example.com/codeflow?start=17&length=80)

Tried this in a browser, and the problem is now when the microphone is used directly by the SDK, the microphone light stays after recognizing speech.

Reply:
Corrected, please try now.